### DISCUSSION TITLE
Data Loss and Education Subscription Transfer Request

### DISCUSSION THREAD

**Discussion by vinhveer** [community/community#121063] [GitHub Education] [answered]
### Hi everyone,

Recently, both my computer and phone underwent a complete data reset, leaving me unable to access my account @vinhveerntu. Despite several attempts, I managed to unlink my email and regain access, but unfortunately, my education subscription has been lost.
How can I restore it and is there any other way to contact someone to retrieve it?

Additionally, I would like to transfer to username @vinhveer

**Comment by samus-aran** [comment, marked as answer]
Hi @vinhveer  :wave:!

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.

If you need assistance to restore access I would suggest you reach out to the [billing & accounts](https://support.github.com/contact/account) support team.
